index.html

```diff
@@ -19,5 +19,8 @@
     <img is="avatar-facebook" username="zenorocha" size="96">
     <pre><code>&lt;img is="avatar-facebook" username="zenorocha" size="96"&gt;</code></pre>
 
+    <img is="avatar-skype" username="live:jonatan_29">
+    <pre><code>&lt;img is="avatar-skype" username="live:jonatan_29"&gt;</code></pre>
```

Comment:
Live is part of the username, look this another username: [jonatam.ribeiro](http://api.skype.com/users/jonatam.ribeiro/profile/avatar).

![jonatam.ribeiro](http://api.skype.com/users/jonatam.ribeiro/profile/avatar)